src/ape/types/__init__.py

```diff
@@ -64,6 +65,30 @@
 TopicFilter = List[Union[Optional[HexStr], List[Optional[HexStr]]]]
 
 
+@dataclass
+class ContractFunctionPath:
+    """
+    Useful for identifying a method in a contract.
+    """
+
+    contract_id: str
+    method_id: Optional[str] = None
```

Comment:
I wouldn't use `method_id` because that has a particular contextual meaning which doesn't match here